src/components/Login/accountConnection/index.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  Button,
+  Flex,
+  Icon,
+  Link,
+  ModalBody,
+  ModalFooter,
+  ModalHeader,
+  Text,
+} from "@chakra-ui/react";
+import React, { Dispatch, SetStateAction } from "react";
+import MainLogoSvg from "@/assets/svg/CourtCanva-main-LOGO.svg";
+import { FcGoogle } from "react-icons/fc";
+import ModalOperator from "@/components/Login/ModalOperater";
+import { api } from "@/utils/axios";
+import { GoogleLoginRes } from "@/components/Login/SelectLogin";
+import { updateUserInfo } from "@/store/reducer/userSlice";
+import { useStoreDispatch } from "@/store/hooks";
+
+interface Props {
+  existedUserInfo: GoogleLoginRes | null;
+  setStep: Dispatch<SetStateAction<number>>;
+  updateLoginData: (data: GoogleLoginRes) => void;
+  onClose: () => void;
+}
+
+const AccountConnection: React.FC<Props> = ({
+  existedUserInfo,
+  setStep,
+  updateLoginData,
+  onClose,
+}) => {
+  const dispatch = useStoreDispatch();
+  const onConnect = async () => {
+    const { data } = await api("/user/connect", {
+      method: "put",
+      requestData: {
+        googleId: existedUserInfo?.googleId,
+        email: existedUserInfo?.email,
+      },
+    });
+    try {
+      if (data) {
+        localStorage.setItem("UserInfo", JSON.stringify(data));
+        dispatch(updateUserInfo(data));
+        updateLoginData(data);
+        setStep(1);
```

Comment:
Maybe you should define step number as enum cause reviewers or other developers don't know what "step 1" means. What do you think？ @tatoMa

Reply:
But this could be solved in next pr.No blocks in this pr.

Reply:
> But this could be solved in next PR.No blocks in this PR.

understood! Thanks for reminding me of that!